client/ClientCommandManager.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * ClientCommandManager.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#ifndef VCMI_IOS
+class PlayerColor;
```

Comment:
Should in theory fix ios compile
```suggestion
VCMI_LIB_NAMESPACE_BEGIN
class PlayerColor;
VCMI_LIB_NAMESPACE_END
```